### DISCUSSION TITLE
can't see stimulate software log in the terminal

### DISCUSSION THREAD

**Discussion by stacyhaha** [snl-dakota/dakota#32] [Q&A]
Dear dakota users:

     I'm using dakota inferface function to connect it with a stimulate software.

     I use python scripts to run the software, just like subprocess.Popen("xxxx")

     What I found is confusing is that the log of the software can't show in the terminal ,while I run the software in terminal, the display of log is expected. I have tried several methods, but still can't taggle with the problem. 

     Hope for your help sincerely~

**Comment by jadamstephens** [comment]
Hi,

Dakota doesn't do anything to redirect the input or output streams of a driver. It's hard to guess what might be going wrong based on your description. Is it possible for you to post your driver?
